app/assets/javascripts/discourse/components/post-menu.js.es6

```diff
@@ -349,6 +349,25 @@ const PostMenuComponent = Ember.Component.extend(StringBuffer, {
     this.sendAction('toggleBookmark', post);
   },
 
+  // Wiki button
+  buttonForWiki(post) {
+    if (!post.get('can_wiki')) return;
+
+    let buttonClass = 'wiki',
+        tooltip = 'post.controls.wiki';
```

Comment:
I think we can move this into an `else` block below so that we don't have to assign the variable twice if the post is a wiki.  :smile: